util/expensivequery/expensivequery.go

```diff
@@ -57,10 +58,17 @@ func (eqh *Handle) Run() {
 		case <-ticker.C:
 			processInfo := sm.ShowProcessList()
 			for _, info := range processInfo {
+				if info.CurTxnStartTS != 0 {
+					txnCostTime := time.Since(info.CurTxnCreateTime)
+					if time.Since(info.ExpensiveTxnLogTime) > 60*time.Second && txnCostTime >= time.Second*time.Duration(threshold) && log.GetLevel() <= zapcore.WarnLevel {
+						metrics.OngoingTxnDurationHistogram.Observe(txnCostTime.Seconds())
```

Comment:
This metric seems little weired to me. It will observe same long-running trx every 60s. So the meaning of the metric is: the runtime distribution of all processes at a given time? The bucket setting `ExponentialBuckets(1, 2, 18)` is unreasonable because `txnCostTime` must be greater than `threshold` (60s by default), that is small buckets like [0,1], (1, 2], (2, 4], ..., (16, 32] would never be used.

Here are some other suggestions:
- it would be better to distinguish user processs from sys processes.
- we reuse query `threshold` here, however it might be too small for a trx, which may lead to annoying logs.